vicky/src/bin/vicky/locks.rs

```diff
@@ -27,6 +34,23 @@ pub async fn locks_get_poisoned_machine(
     locks_get_poisoned(&db).await
 }
 
+
+#[get("/poisoned_detailed")]
```

Comment:
How about using a query parameter here?

`/poisoned?detailed=1` for example?

Reply:
Oh no, these endpoints return two different types of lock structs and are not really compatible. I was thinking that too but i feel like this is more RESTy. Maybe just me tho?